local_modules/execer/execer.cc

```diff
@@ -15,17 +15,56 @@
 #include <dirent.h>
 #include <errno.h>
 #include <fcntl.h>
+#include <time.h>
 #include <node.h>
+#include <sstream>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string>
 #include <sys/socket.h>
+#include <sys/stat.h>
 #include <unistd.h>
 #include <v8.h>
 #include <vector>
 
 using namespace v8;
 
+bool write_response(int fd) {
+	// This is what Nginx does.
+	char buf[2048];
```

Comment:
The size of this doesn't match the error below.

Additionally, the date format is fixed-length [1], so just make this the exact size needed.

[1] https://tools.ietf.org/html/rfc7231#section-7.1.1.1